<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StagePlanSign</class>
 <widget class="QDialog" name="StagePlanSign">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>420</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>420</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>640</width>
    <height>420</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>StagePlanSign</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1,0">
   <property name="spacing">
    <number>8</number>
   </property>
   <property name="leftMargin">
    <number>8</number>
   </property>
   <property name="topMargin">
    <number>8</number>
   </property>
   <property name="rightMargin">
    <number>8</number>
   </property>
   <property name="bottomMargin">
    <number>8</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>8</number>
     </property>
     <property name="leftMargin">
      <number>32</number>
     </property>
     <property name="rightMargin">
      <number>32</number>
     </property>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="minimumSize">
        <size>
         <width>72</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>72</width>
         <height>24</height>
        </size>
       </property>
       <property name="text">
        <string>调度员姓名</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="nameLabel">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>24</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>24</height>
        </size>
       </property>
       <property name="text">
        <string>最近接收时间</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="timeLabel">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>24</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QFrame" name="tableFrame">
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="signBtn">
       <property name="minimumSize">
        <size>
         <width>72</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>72</width>
         <height>24</height>
        </size>
       </property>
       <property name="text">
        <string>签收</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="minimumSize">
        <size>
         <width>72</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>72</width>
         <height>24</height>
        </size>
       </property>
       <property name="text">
        <string>打印</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="closeBtn">
       <property name="minimumSize">
        <size>
         <width>72</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>72</width>
         <height>24</height>
        </size>
       </property>
       <property name="text">
        <string>关闭</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
